src/components/ContributorThumbnail.js

```diff
@@ -189,19 +195,38 @@ const Thumbnail = ({
   } else if (organization.depth === 2) {
     thumbnailImageStyle = classes.thumbnailImage;
   }
+
+  const showtotalChildCount = () => {
+
+    let displayedCount = '';
+
+    if (dropdownLength)
+    {
+      if (filtersActive)
+      {
+        displayedCount = `(${dropdownLength} / ${organization.totalCount})`;
+      }
+      else
+      {
+        displayedCount = `(${dropdownLength})`;
+      }
+    }
+    return displayedCount;
+  };
+
```

Comment:
Correct me if I'm wrong as I haven't spent as much time with the codebase recently, but I recommend rewriting this function with some lines closer, and opening brackets next to their if statement.

```suggestion

  const showtotalChildCount = () => {
    let displayedCount = '';
    
    if (dropdownLength) {
      if (filtersActive) {
        displayedCount = `(${dropdownLength} / ${organization.totalCount})`;
      } else {
        displayedCount = `(${dropdownLength})`;
      }
    }
    return displayedCount;
  };

```